packages/atdgen-runtime/atdgen-runtime.2.0.0/opam

```diff
@@ -18,9 +18,9 @@ build: [
 
 depends: [
   "ocaml" {>= "4.02.3"}
-  "jbuilder" {>= "1.0+beta7"}
+  "jbuilder" {>= "1.0+beta17"}
   "biniou" {>= "1.0.6"}
-  "yojson" {>= "1.2.1"}
+  "yojson" {>= "1.2.1" & < "2.0.0"}
```

Comment:
```suggestion
  "yojson" {>= "1.7.0" & < "2.0.0"}
```

1.2.1 is missing `Yojson.Safe.Util` but I am not entirely sure when this was added (it is not mentioned in the changelog). Given all the other atdgen-runtime versions use `1.7.0` as minimum constraint I feel like that's probably an ok boundary.

Reply:
I looked in the sources, it was introduced in 1.3.1